src/CommandingOfficers/Commander.java

```diff
@@ -300,17 +300,17 @@ public int calculateCombatCharge(UnitDelta minion, UnitDelta enemy, boolean isCo
     if( minion == null || enemy == null )
       return 0;
 
-    int myHPLoss  = minion.getHPDamage();
-    int myHPDealt = enemy.getHPDamage();
+    int guiHPLoss  = minion.getHPDamage() / 10;
+    int guiHPDealt =  enemy.getHPDamage() / 10;
 
     int power = 0; // value in funds of the charge we're getting
 
     // Add up the funds value of the damage done to both participants.
-    power += myHPLoss / UnitModel.MAXIMUM_HP * minion.unit.getCost();
+    power += guiHPLoss * minion.unit.getCost() / 10;
```

Comment:
Is this `10` the assumed maximum HP value, or the conversion factor between health and HP, or something different? In any case, there's a better way to represent this.

Reply:
10 is the assumed maximum *GUI* HP value, and it's GUI HP because it was just converted above.
I could multiply by 10 and then divide by `UnitModel.MAXIMUM_HP`, but bleeeeeh.